lib/simulator-xcode-8.js

```diff
@@ -282,6 +284,16 @@ class SimulatorXcode8 extends SimulatorXcode7 {
    * @throws {Error} If there was an error while setting the location
    */
   async setGeolocation (latitude, longitude) {
+    try {
+      await fs.which(LYFT_SET_LOCATION);
+      await exec(LYFT_SET_LOCATION, [
+        '-c', latitude, longitude,
+        '-u', this.udid
+      ]);
+      return true;
+    } catch (ign) {}
+    log.warn(`Cannot set geolocation with ${LYFT_SET_LOCATION}, because it is not installed or failed to set.`);
```

Comment:
I'd rather log the warning inside `catch` block and additionally print the stderr output. `fs.which` should be in separate block, so we could supply more detailed instructions on how to install the tool if it is not installed

Reply:
it would be also more logical to extract the whole stuff to a separate method/helper function